applications/plugins/hid_app/views/hid_keyboard.c

```diff
@@ -140,18 +140,18 @@ const HidKeyboardKey hid_keyboard_keyset[ROW_COUNT][COLUMN_COUNT] = {
         {.width = 1, .icon = &I_ButtonRight_4x7, .value = HID_KEYBOARD_RIGHT_ARROW},
     },
     {
-        {.width = 3, .icon = NULL, .key = "Ctrl", .value = HID_KEYBOARD_L_CTRL},
+        {.width = 2, .icon = NULL, .key = "Ctl", .value = HID_KEYBOARD_L_CTRL},
```

Comment:
Not enough space to draw them properly. I'll ask our UI/UX team to make proper icons.

Reply:
> Not enough space to draw them properly. I'll ask our UI/UX team to make proper icons.

Okay, Ill wait for icons and will make changes to code after that